architecture.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Justin Hernandez</title>
+    <link rel="stylesheet" href="css/style.css" media="screen" type="text/css" />
+
+  </head>
+
+  <body>
+    <header>
+        <div class="logo">
+          <h1><a href="#">Justin Hernandez</a></h1>
+        </div> <!-- End Logo -->
+        <div class="main__nav">
+          <ul>
+            <li class="listorder--float" id="showSecondaryNav__Design">
+              <a href="#">Design Work
+              <img src="images/arrow_down.svg" class="arrow"></a>
+            </li>
+
+            <li class="listorder--float hide" id="hideSecondaryNav__Design">
+              <a href="#">Design Work<img src="images/arrow_up.svg" class="arrow"></a>
+            </li>
```

Comment:
This is a good idea - thinking about the open and closed state so that the arrows are going the right direction. Do you really need to duplicate all this markup though? Maybe you could use just one set, and use jQuery or CSS to animate the up arrow into a down arrow (by rotating it 180 degrees). 

Try condensing this code!